tests/eval/test_nlp_metrics.py

```diff
@@ -84,8 +92,15 @@ def test_in_context_learning_qa_cot_accuracy():
     assert metric.compute() == (2 / 4)
 
 
+from transformers import BatchEncoding
+
+
+def typed_batch_encoding_access(b: BatchEncoding, k: str) -> Any:
```

Comment:
nit: comment explaining why and when we introduced this

Reply:
oh shoot, this is unused, was just testing things, will delete